### DISCUSSION TITLE
Not able to install the headlessui in Nextjs

### DISCUSSION THREAD

**Discussion by chethanmv-31** [tailwindlabs/headlessui#3431] [Help]
I am trying to install a headless unit in my NextJS project, but I am getting an error. Could you please help me fix this? Please refer to the image below.
![Screenshot 2024-08-20 084601](https://github.com/user-attachments/assets/ff24c98a-a14c-4eb7-a88d-454accbc3583)

**Comment by sanjayatony** [comment]
I have same issue
